### DISCUSSION TITLE
Missing figure numbers

### DISCUSSION THREAD

**Discussion by wiraki** [quarto-dev/quarto-cli#6875] [Q&A] [answered]
### Description

Hello,

This should be a quick question but I was not able to find a reply. Sometimes. when rendering a qmd to html, the "Figure X. " part is missing from the figure caption. The caption text is there and is correct, but the initial figure and number is not there.

What determines this behavior, so that I can fix my document? Last time this happened I had realized I had not included a `#| label:` in each code block that generated an image and that fixed it, but now I am pretty sure there is a `#| label:` directive every time but the figures are still not numbered.

I guess there is something obvious I am missing but I can't figure it out.

Thanks!

**Comment by wiraki** [comment]
Well, I just figured it out. 

Label must start with `fig-` like so:

`#| label: fig-xxx`

**Reply by cscheid** [reply, marked as answer]
This is described here: https://quarto.org/docs/authoring/cross-references.html ("Note that cross reference identifiers must start with their type (e.g. `fig-` or `tbl-`).")
